src/api/controllers/adminPermissionsController.js

```diff
@@ -207,4 +211,155 @@ export default class adminPermissions {
       }
     });
   }
+
+  /**
+   * fetch all reported Articles
+   *
+   * @static
+   * @param {array} req - request array
+   * @param {array} res - respond array
+   * @returns {array} response body
+   * @memberof getArticles
+   */
+  static async getReportedArticles(req, res) {
+    const defaultOffset = 0;
+    const defaultLimit = 10;
+    const offset = req.query.offset || defaultOffset;
+    const limit = req.query.limit || defaultLimit;
+    const paginatedArticle = await Report.findAll({
+      attributes: [['createdAt', 'ReportedOn']],
```

Comment:
@Ramadhan0 why exactly are you changing the `createdAt` attribute name? Have we been doing that across other API endpoints? Let's be consistent.